CIME/hist_utils.py

```diff
@@ -609,13 +609,14 @@ def generate_teststatus(testdir, baseline_dir):
         )
 
 
-def _generate_baseline_impl(case, baseline_dir=None, allow_baseline_overwrite=False):
+def _generate_baseline_impl(case, baseline_dir=None, allow_baseline_overwrite=False, allow_baseline_skip=False):
     """
     copy the current test output to baseline result
 
     case - The case containing the hist files to be copied into baselines
     baseline_dir - Optionally, specify a specific baseline dir, otherwise it will be computed from case config
-    allow_baseline_overwrite must be true to generate baselines to an existing directory.
+    allow_baseline_overwrite must be true to generate baselines to an existing directory. allow_baseline_skip
```

Comment:
It looks like "allow_baseline_skip" ended up on the line with "allow_baseline_overwrite" rather than starting the new line.

Reply:
I removed all changes from this file, so resolving.